thirdparty/faiss/faiss/utils/distances.cpp

```diff
@@ -211,30 +214,59 @@ void exhaustive_inner_product_seq_impl(
     using SingleResultHandler = typename BlockResultHandler::SingleResultHandler;
     int nt = std::min(int(nx), omp_get_max_threads());
 
+#ifdef FAISS_WITH_DNNL
```

Comment:
@alexanderguzhva The filter is inside Knowhere or in the Milvus?

Reply:
@xtangxtang an external filter (in the form of bitset), provided from Milvus

Reply:
@alexanderguzhva Thanks for your clarification. Since this PR is just for the Knowhere, The Benchmark for this PR should include the bitset filter? or Just an memory bandwitdth perf data is OK?